compiler/src/dotty/tools/dotc/typer/Implicits.scala

```diff
@@ -1174,9 +1174,7 @@ trait Implicits { self: Typer =>
         if (ownerScore < 0) return false
         val arity1 = sym1.info.firstParamTypes.length
         val arity2 = sym2.info.firstParamTypes.length
-        if (arity1 < arity2) return true
-        if (arity1 > arity2) return false
-        false
+        arity1 < arity2
```

Comment:
These two pieces of code look equivalent to me, if `arity1 < arity2` we return true, otherwise we return false.